fastlane/lib/fastlane/fast_file.rb

```diff
@@ -313,8 +314,9 @@ def import_from_git(url: nil, branch: 'HEAD', path: 'fastlane/Fastfile', version
               # so we can switch between tags or branches instantly, or else,
               # it would defeat the caching's purpose.
               depth = is_eligible_for_caching ? "" : "--depth 1"
-
-              Actions.sh("git clone #{url.shellescape} #{clone_folder.shellescape} #{depth} --no-checkout #{branch_option}")
+              command = "git clone #{url.shellescape} #{clone_folder.shellescape} #{depth} --no-checkout #{branch_option}"
+              command << " -c http.extraheader='Authorization: Basic #{git_basic_authorization}'" unless git_basic_authorization.nil?
```

Comment:
To improve the readability and maintainability of this code, please make it explicit that it's a configuration setting:

```suggestion
              command << " --config http.extraheader='Authorization: Basic #{git_basic_authorization}'" unless git_basic_authorization.nil?
```

https://git-scm.com/docs/git-clone#Documentation/git-clone.txt-code-ccodeemltkeygtemcodecodeemltvaluegtem